src/convert.ts

```diff
@@ -0,0 +1,13 @@
+import { format, compareAsc } from "date-fns";
+
+export function convertSelicEntry(data: string, valor: string) {
+  try {
+    const date = new Date(format(format(data, "dd/MM/yyyy"), "yyyy-MM-dd"));
+    const value = Number(valor);
+    if (isNaN(value))
+      return null;
+    return { date, value };
+  } catch (error) {
+    return null;
+  }
```

Comment:
I'd rather us to throw exceptions to let the dev know that the conversion failed, opposite to failing silently.